client/data/help/use-help-search-query.ts

```diff
@@ -66,8 +74,11 @@ const fetchArticlesAPI = async (
 			path: `/help-center/search?${ queryString }`,
 		} as APIFetchOptions ) ) as SearchResult[];
 	}
+	// Remove articles that are already in the tailored articles.
+	const searchResult = filterOutDuplicatedItems( articlesResponse, searchResultResponse );
```

Comment:
Pay attention that this is confusing. You pass articles and then search, while the function parameters are search and then articles.
I spent a little trying to understand why it worked, before noticing this swap 😄